pkg/collectors/replicaset.go

```diff
@@ -107,19 +115,20 @@ type replicasetCollector struct {
 }
 
 // Describe implements the prometheus.Collector interface.
-func (dc *replicasetCollector) Describe(ch chan<- *prometheus.Desc) {
+func (rs *replicasetCollector) Describe(ch chan<- *prometheus.Desc) {
 	ch <- descReplicaSetCreated
 	ch <- descReplicaSetStatusReplicas
 	ch <- descReplicaSetStatusFullyLabeledReplicas
 	ch <- descReplicaSetStatusReadyReplicas
 	ch <- descReplicaSetStatusObservedGeneration
 	ch <- descReplicaSetSpecReplicas
 	ch <- descReplicaSetMetadataGeneration
+	ch <- descReplicaSetOwner
 }
 
 // Collect implements the prometheus.Collector interface.
-func (dc *replicasetCollector) Collect(ch chan<- prometheus.Metric) {
-	rss, err := dc.store.List()
+func (rs *replicasetCollector) Collect(ch chan<- prometheus.Metric) {
```

Comment:
Small nit, looking at the other controllers this should be called `rc` or `rsc`, right?

Reply:
Good catch. `rsc` should be fine.

Reply:
Done. PTAL. @mxinden